mlocal/frags/Makefile.stub

```diff
@@ -93,16 +93,26 @@ actions_LIST := $(SOURCEDIR)/etc/actions/exec $(SOURCEDIR)/etc/actions/run $(SOU
 	$(SOURCEDIR)/etc/actions/start $(SOURCEDIR)/etc/actions/test
 actions_INSTALL := $(DESTDIR)$(SYSCONFDIR)/singularity/actions
 
-INSTALLFILES := $(singularity_INSTALL) $(starter_INSTALL) $(starter_suid_INSTALL) $(sessiondir) \
-	$(config_INSTALL) $(dist_bin_SCRIPTS_INSTALL) $(capability_JSON) $(syecl_config_INSTALL) \
-	$(bash_completion_INSTALL) $(actions_INSTALL) $(cni_plugins_INSTALL) $(cni_config_INSTALL) \
-	$(seccomp_profile_INSTALL) $(NVIDIA_liblist_INSTALL) $(cgroups_config_INSTALL) 
+CLEANFILES += $(singularity) $(go_BIN) $(go_OBJ) $(bash_completion)
```

Comment:
Setting up the default variables for the limited non-runtime values and then adding other targets below if we are in fact building the runtime.